cmux-tui/bindings/python/cmux/client_defaults.py

```
@@ -1,18 +1,64 @@
+        raise ValueError(
+            "session name must be a non-empty path component "
+            "without separators or control characters"
+        )
+
+
 def default_socket_path(session: str = "main") -> str:
+    """Return the default socket path after validating the session name."""
+    validate_session_name(session)
     runtime = os.environ.get("XDG_RUNTIME_DIR")
     if not runtime:
         runtime = os.environ.get("TMPDIR") or "/tmp"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)